config-wdio/local.conf.ts

```diff
@@ -0,0 +1,37 @@
+var defaults = require("./base.conf").config;
+var _ = require("lodash");
+import * as configVal from "config";
+const BASEURL = configVal.get("Environment.baseUrl");
+const BROWSER = process.env.BROWSER||'chrome';
+let DRIVERS;
```

Comment:
Would you please either declare this variable as Const or change the DRIVERS to small case.

Reply:
done, changed it drivers